aws/createTemplate.sh

```diff
@@ -718,6 +713,37 @@ function process_template() {
       ;;
   esac
 
+  # Handle >=v2.0.1 cmdb where du/placement subdirectories were introduced for state
+  #
+  # Assumption is that if files whose names contain the du are not present in
+  # the base cf_dir, then the du directory structure should be used
+  #
+  # This will start to cause the new structure to be created by default for new units,
+  # and will accommodate the cmdb update when it is performed.
+  case "${level}" in
+    unitlist|blueprint|buildblueprint)
+      # No subdirectories for deployment units
+      ;;
+    *)
+      readarray -t legacy_files < <(find "${cf_dir}" -mindepth 1 -maxdepth 1 -name "*${deployment_unit}*" )
+
+      if [[ (-d "${cf_dir_default}/${deployment_unit}") || "${#legacy_files[@]}" -eq 0 ]]; then
+        case "${region}" in
+          us-east-1)
+            local placement="global"
```

Comment:
I think this would need to be further filtered for cert only. We have some actual deployments to us-east-1 which are part of the default placement

Reply:
@roleyfoley it is also for lambda at edge. What are the examples where the placement should be "default" when us-east-1 is targetted?